src/test/kotlin/com/terraformation/backend/db/GeometrySerializerTest.kt

```diff
@@ -74,56 +97,69 @@ internal class GeometrySerializerTest {
       return Stream.of(
           arguments(
               "Point with long/lat SRID",
-              """{"type":"Point","coordinates":[1.1,2.2,3.3]}""",
-              "SRID=4326;POINT(1.1 2.2 3.3)"),
+              """{"type":"Point","coordinates":[1.1,-2.2,3.3]}""",
+              SRID.LONG_LAT,
+              "POINT(1.1 -2.2 3.3)"),
           arguments(
               "Point with Mercator SRID",
               """{"type":"Point",$mercatorCrs,"coordinates":[1.1,2.2,3.3]}""",
-              "SRID=3857;POINT(1.1 2.2 3.3)"),
+              SRID.SPHERICAL_MERCATOR,
+              "POINT(1.1 2.2 3.3)"),
           arguments(
               "MultiPoint with long/lat SRID",
               """{"type":"MultiPoint","coordinates":[[1,1,1],[2,2,2]]}""",
-              "SRID=4326;MULTIPOINT(1 1 1,2 2 2)"),
+              SRID.LONG_LAT,
+              "MULTIPOINT(1 1 1,2 2 2)"),
           arguments(
               "MultiPoint with Mercator SRID",
               """{"type":"MultiPoint",$mercatorCrs,"coordinates":[[1,1,1],[2,2,2]]}""",
-              "SRID=3857;MULTIPOINT(1 1 1,2 2 2)"),
+              SRID.SPHERICAL_MERCATOR,
+              "MULTIPOINT(1 1 1,2 2 2)"),
           arguments(
               "Polygon with long/lat SRID",
               """{"type":"Polygon","coordinates":[[[1,2,3],[2.1,1.1,0.1],[5,5,5],[1,2,3]]]}""",
-              "SRID=4326;POLYGON((1 2 3,2.1 1.1 0.1,5 5 5,1 2 3))"),
+              SRID.LONG_LAT,
+              "POLYGON((1 2 3,2.1 1.1 0.1,5 5 5,1 2 3))"),
           arguments(
               "MultiPolygon with long/lat SRID",
-              """{"type":"MultiPolygon","coordinates":[[[[1,2,3],[1.1,2.2,3.3],[0,0,0],[1,2,3]],[[0,-1,1],[1,0,1],[0,1,0],[0,-1,1]]]]}""",
-              "SRID=4326;MULTIPOLYGON(((1 2 3,1.1 2.2 3.3,0 0 0,1 2 3),(0 -1 1,1 0 1,0 1 0,0 -1 1)))"),
+              """{"type":"MultiPolygon","coordinates":[[[[1,2,3],[1,2.5,3],[0.0,2.5,3],[1,2,3]]],[[[11,2,13],[11,2.5,13],[10,2.5,13],[11,2,13]]]]}""",
```

Comment:
The previous version of this multipolygon had bad geometry (it specified a hole that fell outside the bounds of the enclosing shape, and it had edges that crossed one another).